### PR TITLE
2017: Rear-View Mirror Reflections

### PR DESCRIPTION
**Pull request by AparnaKarve**
@JPrause Please review this year-end special edition of LWIMIQ.

Thanks.

### CODE REVIEW

**Comment by JPrause** [comment]
Thanks @AparnaKarve 
@imtayadeway @d-m-u @syncrou can each of you also have a look if time permits.

File: site/_posts/2017-12-18-Look-back-and-reflect.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+date: 2017-12-18
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+HELLO Folks! [Aparna](https://github.com/AparnaKarve) here with a special year-end edition for the 'Last Week in ManageIQ'.  Strap on your seat belts as we get into the time portal machine and review what made 2017 special.
+
+
+- The Great Split
```
Comment (imtayadeway) [inline site/_posts/2017-12-18-Look-back-and-reflect.md]: Did you consider using headings as opposed to bullet points for these?

Comment (AparnaKarve) [reply site/_posts/2017-12-18-Look-back-and-reflect.md]: Sure, headings would look better.

File: site/_posts/2017-12-18-Look-back-and-reflect.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+And now...for the Showstopper event... (drum roll here!) 
+
+`Fine` released in May 2017 and `Gaprindashvili` -- scheduled for a GA release early next year.
+
+
+
+Looking ahead, heads down on fixing Gaprindashvli bugs but an eye toward the next Release - code named H-Release or the [Hammer](https://en.wikipedia.org/wiki/Jon_Ludvig_Hammer) Release due for 2018 -  big push on architecture improvements and a vehicle to pull the curtain down on today’s issues.
+
+
+To wrap up, we should feel really good about what we have accomplished this year – we totally deserve that extra slice of cake or the decadent chocolate cookie this holiday season as we curl our feet up by the fire, drink hot cocoa and soak in the much needed R&R – 2018 is going to be even bigger and better – Happy Holidays and a Happy New Year!
```
Comment (imtayadeway) [inline site/_posts/2017-12-18-Look-back-and-reflect.md]: 😋 🍰

**Comment by AparnaKarve** [comment]
Thanks @imtayadeway !
Addressed your above comment.

**Comment by JPrause** [comment]
@AparnaKarve this is excellent! Awesome theme and it's great to refelct back on the last year. Wow, lots happened,....with even more to come in the New Year.

LGTM @cybette can you review as time permits and merge when possible.

**Comment by AparnaKarve** [comment]
Thanks @JPrause @cybette

**Pull request merged**